src/include/firebird/Interface.h

```diff
@@ -183,11 +184,21 @@ namespace Firebird
 			{
 				status->setErrors(e.getStatus()->getErrors());
 			}
-			catch (...)
+			catch (const std::exception& e)
```

Comment:
Would not that add dependency on C++ runtime libraries in the client?

Reply:
At least for MSVC and GCC - no, but it looks like MacOS wants something more.